packages/email/src/templates/partner-application-received.tsx

```
@@ -133,27 +140,16 @@ export default function PartnerApplicationReceived({
-                      <span className="font-normal text-neutral-500">
-                        (optional)
-                      </span>
-                    </Text>
-                    <Text className="m-0 p-0 leading-6 text-neutral-600">
-                      {partner.comments ?? "No comments provided."}
-                    </Text>
-                  </Section>
-
+                  {partner.applicationFormData.map((field) => (
+                    <Section className="mb-6">
+                      <Text className="m-0 mb-2 p-0 text-base font-medium text-neutral-900">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)